modules/network/src/test/resources/simplelogger.properties

```diff
@@ -0,0 +1,52 @@
+#
```

Comment:
This configuration leads to excessive amount of logs (courtesy of scalecube)

Reply:
Don't see any problems here. It's ok to have logs for analysis in tests.

Reply:
Ok